command/genesis/polybft_params.go

```diff
@@ -289,6 +294,61 @@ func (p *genesisParams) deployContracts(totalStake *big.Int) (map[types.Address]
 			artifact: contractsapi.L2StateSender,
 			address:  contracts.L2StateSenderContract,
 		},
+		{
+			artifact: contractsapi.ValidatorSet,
+			address:  contracts.NewValidatorSetContract,
+			constructorCallback: func(artifact *artifact.Artifact,
+				polybftConfig *polybft.PolyBFTConfig,
+				rootConfig *polybft.RootchainConfig) ([]byte, error) {
+				// TODO @goran-ethernal- this will be changed in the next PRs to use generated binding
+				// once we remove the old ChildValidatorSet on use the new ValidatorInit binding
+				validatorsMap := make([]map[string]interface{}, len(polybftConfig.InitialValidatorSet))
+				for i, validator := range polybftConfig.InitialValidatorSet {
+					validatorsMap[i] = map[string]interface{}{
+						"addr":  validator.Address,
+						"stake": validator.Stake,
+					}
+				}
+
+				// TODO @goran-ethernal - we will remove once we change e2e tests
+				// since by RFC-201 we won't be able to start edge without bridge (root)
+				customSupernetManagerAddr := types.ZeroAddress
+				if rootConfig != nil {
+					customSupernetManagerAddr = rootConfig.CustomSupernetManagerAddress
+				}
```

Comment:
Change this and we can remove `*polybft.RootchainConfig` param from the function signature alltogether. WDYT?
```suggestion
				if polybftConfig.Bridge != nil {
					customSupernetManagerAddr = polybftConfig.Bridge.CustomSupernetManagerAddr
				}
```

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1368/commits/7cec43f8fd68498c9589fe0c4867df9890532f78